.circleci/config.yml

```diff
@@ -152,6 +142,8 @@ jobs:
             if [ $CIRCLE_PROJECT_USERNAME = 'palantir' ] && [ -z $CIRCLE_PR_NUMBER ] && [ $CIRCLE_TAG ]; then
               ./scripts/circle-ci/publish-images-to-antithesis.sh
             fi
+      - run:
+          command: ./scripts/circle-ci/trigger-antithesis-simulation.sh
```

Comment:
We don't need to spin up a new container as suggested by devtools oncall:

https://palantir.slack.com/archives/C2YPL1MB7/p1726664710293049